### PR TITLE
Feat/enable custom element

### PR DESCRIPTION
**Pull request by mshaaban0**
## What's Changing
- Fix readme example and indicate what is "Pane" component
- Add `customElement` prop to enable changing the wrapper html element type
- Fix prev, and next buttons semantic a11y roles

## Change Type

- [x] `documentation`
- [x] `minor`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,24 +48,30 @@ To get access to the current glider instance this react component exposes a ref.
...
+
+const PaneExample: React.FC<PaneProps> = ({ children, style, className }) => (
+  <div className={`glider-slide ${className}`} style={style}>
+    <h1>{children}</h1>
+  </div>
+);
 
 const example = () => {
-  const gliderRef = React.useRef<GliderMethods>();
+  const gliderRef = React.useRef < GliderMethods > null;
```
Comment (hipstersmoothie) [inline README.md:60]: this should be a function call still

File: src/index.tsx [hunk 2]
```diff
@@ -269,10 +274,17 @@ const GliderComponent = React.forwardRef(
...
     return (
-      <div className="glider-contain">
+      <Element className="glider-contain">
         {props.hasArrows && !props.arrows && (
-          <button role="button" className="glider-prev" id="glider-prev">
+          <button
+            type="button"
+            className="glider-prev"
+            aria-label="Previous"
+            id="glider-prev"
```
Comment (hipstersmoothie) [inline src/index.tsx:286]: Can you use @react/auto-id to create the ID? that way if a user uses multiple gliders on a page there won't be duplicate IDs

Comment (mshaaban0) [reply src/index.tsx:286]: hmm, I actually ran into that problem.
I think adding another lib will cost us some extra kilobytes, how about adding a prop to prefix all ids, i.e. `prefixId="homeSlider"` which will result to `id="homeSlider-glider-prev"` ?

Comment (hipstersmoothie) [reply src/index.tsx:286]: the lib is pretty small. I think the kbs are better than another prop

https://bundlephobia.com/result?p=@reach/auto-id@0.12.1

**Review by hipstersmoothie — APPROVED** [review]

**Comment by hipstersmoothie** [comment]
:rocket: PR was released in [`v2.1.0`](https://github.com/hipstersmoothie/react-glider/releases/tag/v2.1.0) :rocket:

**Pull request merged**